adminapi/templates/base_inner.html

```diff
@@ -0,0 +1,22 @@
+<html>
+    <head>
+        <title>Login</title>
```

Comment:
Don't assume anything in base_inner.html. Make it a {% block title %}{% endblock %} and have other templates fill it.